src/jit/lclvars.cpp

```diff
@@ -2215,7 +2215,11 @@ void Compiler::lvaSetVarDoNotEnregister(unsigned varNum DEBUGARG(DoNotEnregister
 // Returns true if this local var is a multireg struct
 bool Compiler::lvaIsMultiregStruct(LclVarDsc* varDsc)
 {
+#if defined(_TARGET_ARM64_)
+    if (varTypeIsStruct(varDsc->TypeGet()))
```

Comment:
I think this change should be made unconditionally. It is not called for amd64 except in the case that it has already handled its non-TYP_STRUCT structs (i.e. SIMD).